src/Button.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 <label role="checkbox"
        :class="className"
-       :aria-checked="ariaChecked">
+       :aria-checked="!!toggled">
```

Comment:
`aria-checked` requires an actual word "true" or "false". This introduces a bug.

Reply:
No it doesn't, I tested it. This coerces the value to a boolean, and the boolean then gets rendered as a string by Vue.

Reply:
Example:

https://jsfiddle.net/63hjbL5p/1/ - look in inspector.

Requirement: 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_checkbox_role

```
aria-checked="true": The checkbox is checked
aria-checked="false": The checkbox is not checked
aria-checked="mixed": The checkbox is partially checked
```

```
Example:
<span role="checkbox" aria-checked="false" tabindex="0" id="chk1"></span>
```

Reply:
Yes, I'm sorry, it does get removed you're right. I'll adjust it.